lib/init.coffee

```diff
@@ -6,13 +6,13 @@ module.exports =
     executablePath:
       title: 'Executable Path'
       type: 'string'
-      default: 'pep257'
-      description: "Path to executable pep257 cmd."
+      default: 'pydocstyle'
```

Comment:
Hmm, I just tried out pip-uninstalling `pydocstyle` and starting up Atom with this version of the package, and I get a good warning already. Also when I do have `pydocstyle` installed and pass impossible command line option via the settings menu, I get a good warning. This is on a Linux machine though, I had encountered the problem on a Windows machine. I'll open an issue for that and will look into it when I'm back on the Windows machine, but I'd agree to merge this one for now.